src/components/ClusterNodes/ColumnSelector.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState, useEffect } from 'react'
+import ReactSelect, { components, MultiValue } from 'react-select'
+import { Option } from '../common'
+import { COLUMN_METADATA, ColumnMetadataType } from './types'
+import { ReactComponent as Setting } from '../../assets/icons/ic-nav-gear.svg'
+import { containerImageSelectStyles } from '../CIPipelineN/ciPipeline.utils'
+
+const ValueContainer = (props: any): JSX.Element => {
```

Comment:
We should move this function `ValueContainer`  in utils file

Reply:
It is only used in one place

Reply:
Yes. ValueContainer impl may differ for different places so can be kept in respective files. We also have a common comp if we want to use it at some places.